tests/unit/test_attribute_configuration.py

```diff
@@ -57,22 +59,56 @@ def test_resource_from_json(self):
 
     @responses.activate
     def test_delete(self):
-        base = "http://localhost:9100/api/versioned/v1"
-        alias = "projects/1/attributeConfigurations"
-        attribute_id = "26"
-
-        url = f"{base}/{alias}/{attribute_id}"
+        url = f"{self._base}/{self._alias}/{self._attribute_id}"
         responses.add(responses.GET, url, json=self._ac_json)
         responses.add(responses.DELETE, url, status=204)
         responses.add(responses.GET, url, status=404)
 
-        collection = AttributeConfigurationCollection(self.unify, alias)
-        config = collection.by_resource_id(attribute_id)
+        collection = AttributeConfigurationCollection(self.unify, self._alias)
+        config = collection.by_resource_id(self._attribute_id)
         self.assertEqual(config._data, self._ac_json)
 
         response = config.delete()
         self.assertEqual(response.status_code, 204)
-        self.assertRaises(HTTPError, lambda: collection.by_resource_id(attribute_id))
+        self.assertRaises(
+            HTTPError, lambda: collection.by_resource_id(self._attribute_id)
+        )
+
+    @responses.activate
+    def test_update(self):
+        def create_callback(request, snoop):
+            snoop["payload"] = request.body
+            return 200, {}, json.dumps(self._updated_ac_json)
+
+        configs_url = f"{self._base}/{self._alias}"
+        config_url = f"{configs_url}/{self._attribute_id}"
+
+        snoop_dict = {}
+        responses.add(responses.GET, config_url, json=self._ac_json)
+        responses.add_callback(
+            responses.PUT, config_url, partial(create_callback, snoop=snoop_dict)
+        )
+        configs = AttributeConfigurationCollection(self.unify, self._alias)
+        config = configs.by_resource_id(self._attribute_id)
+
+        temp_spec = config.spec().with_attribute_role("SUM_ATTRIBUTE")
+        new_config = (
+            temp_spec.with_enabled_for_ml(False)
```

Comment:
nit: fluent style wants even the first `.with_*` on a separate line:
```
    temp_spec
        .with_enabled_for_ml(False)
        .with_similarity_function("ABSOLUTE_DIFF")
        .with_tokenizer("BIGRAM")
        .put()
```

Reply:
Black, however, does not.